tests/inject-live-reload-test.js

```diff
@@ -95,6 +95,15 @@ describe('dynamicScript returns right script when', hooks => {
     assert.strictEqual(actual, 'http://localhost:35729/_lr/livereload.js?port=35729&host=localhost&path=_lr/livereload');
   });
 
+  it('liveReloadBaseUrl is provided', assert => {
```

Comment:
Can you append `GH#68` to the test name (so we can more easily remember the original issue)?

Reply:
Done!